doc/userguide/devguide/contributing/backports-guide.rst

```diff
@@ -0,0 +1,107 @@
+========================
+Suricata Backports Guide
+========================
+
+This document describes the processes used to backport content to previous
+Suricata releases. Most often, this means security and/or bug fixes;
+however, in some cases, features may be backported to previous Suricata releases.
+
+There are multiple versions of Suricata at any given time:
+    * Master
+    * Current release
+    * Release prior to current release
+
+For example, at the moment, there are 3 releases based on these Suricata branches:
+    * master: 8.0.0-dev, current development branch
+    * main-7.0.x: current stable release (note we're changing our naming conventions)
+    * master-6.0.x: previous stable release
```

Comment:
Do we plan on upgrading this doc everytime we EOL a branch or create a new one?

Reply:
Good question. Considering it's something to be done every... two or three years, I don't think it's a bad idea to review/ check for needed updates at least then. imho.